.github/CHANGELOG.md

```diff
@@ -78,8 +80,8 @@
   [(#112)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/112)
 
   This new feature empowers users to leverage the computational power of multi-node and multi-GPUs for running large-scale applications. It requires both the total number of overall `MPI` processes and the number of `MPI` processes of each node to be the same and power of `2`. Each `MPI` process is responsible for managing one GPU for the moment. 
-  To enable this feature, users can set `mpi=True`. Furthermore, users can fine-tune the performance of `MPI` operations by adjusting the `mpi_buf_size` parameter. This parameter determines the allocation of `mpi_buf_size` MB(megabytes) GPU memory for `MPI` operations. Note that `mpi_buf_size` should be also power of 2 and there will be a runtime warning if GPU memory buffer for MPI operation is larger than the GPU memory allocated for the local state vector. By default (`mpi_buf_size=0`), the GPU memory allocated for MPI operations will be the same of size of the local state vector, with a limit of 64 MB.
-  The workflow for the new feature is:
+  To enable this feature, users can set `mpi=True`. Furthermore, users can fine-tune the performance of `MPI` operations by adjusting the `mpi_buf_size` parameter. This parameter determines the allocation of `mpi_buf_size` MiB (mebibytes) GPU memory for `MPI` operations. Note that `mpi_buf_size` should be also power of 2 and there will be a runtime warning if GPU memory buffer for MPI operation is larger than the GPU memory allocated for the local state vector. By default (`mpi_buf_size=0`), the GPU memory allocated for MPI operations will be the same of size of the local state vector, with a limit of 64 MiB. Note that MiB is different from MB (megabytes). Mebibytes (MiB) are binary units of measurement where 1 MiB is equal to 1,048,576 bytes, while megabytes (MB) are decimal units of measurement where 1 MB is equal to 1,000,000 bytes.
+  The workflow for the new feature is as follows:
   ```python
   from mpi4py import MPI
```

Comment:
Can we use 4 spaces for indenting the Python code here?

Reply:
Done!